web/embed_components.html

```diff
@@ -27,7 +23,7 @@
       <paper-tab name="html">HTML</paper-tab>
       <paper-tab name="css">CSS</paper-tab>
     </paper-tabs>
-    <paper-progress id="edit-progress" class="bottom fit" hidden></paper-progress>
+    <paper-progress id="edit-progress"  class="bottom fit" hidden style="top: inherit"></paper-progress>
```

Comment:
You moved a bunch of elements back to using inline `style`s - was there a reason?